README.md

```diff
@@ -128,6 +128,27 @@ Parameters:
   |-------|-----|------------|---------|--------|
   |get    |lang |en, gb ...  |true     |en      |
 
+###Devlopment tips
+
+If you intend to contribute on the project, those tips might be quite helpful!
+
+####Get list of the item you're currently seeing on the map
+
+If you want to see all the information provided by an item you're currently seeing on the map, you can do it with your web browser development console. Please note that it's **simpler** to do when there are very few objects displayed, so at very high level of zoom is better:
```

Comment:
It could be useful to add a link to mapbox-gl-js docs: 
https://docs.mapbox.com/mapbox-gl-js/api/#map#queryrenderedfeatures